src/app/entry/superDescriptorLanguage.ts

```diff
@@ -0,0 +1,110 @@
+import { DescriptorLanguageBean, SourceFile, ToolDescriptor, Workflow } from 'app/shared/swagger';
+
+/**
+ * Use the value property to map the DescriptorLanguageBean to this
+ */
+export interface SuperDescriptorLanguage extends DescriptorLanguageBean {
```

Comment:
"super" seems funny as a word since superclass is normally the base class. 
In this cases, this seems like an extended descriptor language?